api_tests/search/views/test_views.py

```diff
@@ -135,485 +165,496 @@ def test_search_fields_links_with_query(self):
         components_link = search_fields['components']['related']['href']
         registrations_link = search_fields['registrations']['related']['href']
 
-        assert_in('/{}search/users/?q=science'.format(API_BASE), users_link)
-        assert_in('/{}search/files/?q=science'.format(API_BASE), files_link)
-        assert_in('/{}search/projects/?q=science'.format(API_BASE), projects_link)
-        assert_in('/{}search/components/?q=science'.format(API_BASE), components_link)
-        assert_in('/{}search/registrations/?q=science'.format(API_BASE), registrations_link)
+        assert '/{}search/users/?q=science'.format(API_BASE) in users_link
+        assert '/{}search/files/?q=science'.format(API_BASE) in files_link
+        assert '/{}search/projects/?q=science'.format(API_BASE) in projects_link
+        assert '/{}search/components/?q=science'.format(API_BASE) in components_link
+        assert '/{}search/registrations/?q=science'.format(API_BASE) in registrations_link
 
 
 class TestSearchComponents(ApiSearchTestCase):
 
-    def setUp(self):
-        super(TestSearchComponents, self).setUp()
-        self.url = '/{}search/components/'.format(API_BASE)
+    @pytest.fixture()
+    def url_component_search(self):
+        return '/{}search/components/'.format(API_BASE)
+
+    def test_search_components(self, app, all, user, user_one, user_two, component, component_public, component_private, url_component_search):
 
-    def test_search_public_component_no_auth(self):
-        res = self.app.get(self.url)
-        assert_equal(res.status_code, 200)
+        #test_search_public_component_no_auth(self, app, all, url_component_search, component, component_public):
+        res = app.get(url_component_search)
+        assert res.status_code == 200
         num_results = len(res.json['data'])
         total = res.json['links']['meta']['total']
-        assert_equal(num_results, total, 2)
-        assert_in(self.component.title, res)
-        assert_in(self.component_two.title, res)
+        assert num_results == total, 2
+        assert component_public.title in res
+        assert component.title in res
 
-    def test_search_public_component_auth(self):
-        res = self.app.get(self.url, auth=self.user)
-        assert_equal(res.status_code, 200)
+        #test_search_public_component_auth(self, app, all, url_component_search, user, component, component_public):
+        res = app.get(url_component_search, auth=user)
+        assert res.status_code == 200
         num_results = len(res.json['data'])
         total = res.json['links']['meta']['total']
-        assert_equal(num_results, total, 2)
-        assert_in(self.component.title, res)
-        assert_in(self.component_two.title, res)
+        assert num_results == total, 2
```

Comment:
A quick regex showed this is the only file that uses that pattern, I think it was 38 in here that will need to be converted.